src/app/static/src/app/components/uploadInputs/UploadInputs.vue

```diff
@@ -146,45 +146,45 @@
                     error: state.pjnzError,
                     fromADR: !!state.pjnz?.fromADR,
                     existingFileName: (state.pjnz && state.pjnz.filename) || state.pjnzErroredFile
-                } as PartialFileUploadProps),
+                }),
                 shape: (state: BaselineState) => ({
                     valid: state.shape != null,
                     error: state.shapeError,
                     fromADR: !!state.shape?.fromADR,
                     existingFileName: (state.shape && state.shape.filename) || state.shapeErroredFile
-                } as PartialFileUploadProps),
+                }),
                 population: (state: BaselineState) => ({
                     valid: state.population != null,
                     error: state.populationError,
                     fromADR: !!state.population?.fromADR,
                     existingFileName: (state.population && state.population.filename) || state.populationErroredFile
-                } as PartialFileUploadProps),
+                }),
                 hasBaselineError: (state: BaselineState) => !!state.baselineError,
                 baselineError: (state: BaselineState) => state.baselineError,
                 validating: (state: BaselineState) => state.validating
             }),
-            ...mapState<MetadataState>("metadata", {
+            ...mapState<MetadataState, {"plottingMetadataError": (this: CustomVue, state: MetadataState) => Error | null}>("metadata", {
```

Comment:
This looks strange, what's happening here ?

Reply:
this is jjust defining the interface it returns inside the generic types, I have extracted to an external type for clarity